lib/core/base/context.js

```diff
@@ -128,7 +128,7 @@ function parseSelectorArray(context, type) {
 		if (typeof item === 'string') {
 			result = result.concat(utils.toArray(document.querySelectorAll(item)));
 			break;
-		} else if (item && item.length) {
+		} else if (item && Array.isArray(item)) {
```

Comment:
This test will not allow jQuery objects to work anymore, we should probably be excluding actual nodes here rather than testing for an array